console/src/main/java/org/web3j/console/project/InteractiveImporter.java

```diff
@@ -0,0 +1,22 @@
+package org.web3j.console.project;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+class InteractiveImporter extends InteractiveOptions {
+
+    InteractiveImporter() {
```

Comment:
empty constructor is a code smell... what is the purpose of this?

Reply:
For a more fluid API, since the super default constructor uses System.in and System.out I thought it would look cleaner.